openstef/pipeline/train_create_forecast_backtest.py

```diff
@@ -32,150 +35,51 @@ def train_model_and_forecast_back_test(
     List[pd.DataFrame],
     List[pd.DataFrame],
 ]:
-    """Pipeline for a back test.
```

Comment:
Here you remove the documentation of the `train_model_and_forecast_back_test` , Could you please update the documentation and add it to the updated function?